libultraship/libultraship/Lib/Fast3D/gx2_shader_gen.c

```diff
@@ -656,7 +660,7 @@ static int generateVertexShader(GX2VertexShader *vsh, struct CCFeatures *cc_feat
     );
 
     // params
-    for (int i = 0; i < num_ps_inputs - 1; i++) {
+    for (uint32_t i = 0; i < num_ps_inputs - 1; i++) {
```

Comment:
```suggestion
    for (size_t i = 0; i < num_ps_inputs - 1; i++) {
```
Shouldn't this be size_t